From 032d7227c4abb2fd784a890d9693f6444d3ad59d Mon Sep 17 00:00:00 2001
From: xbw <xbw@rock-chips.com>
Date: Mon, 24 Mar 2014 07:20:28 +0800
Subject: [PATCH] ARM: rockchip:   revert the rk3288 workaround modifyed in
 commit-22fbc55379c9f8ac29a89884ad7863c281bb926d     diff --git
 a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
 index bc3a240..76ddbfd 100755 --- a/arch/arm/configs/rockchip_defconfig +++
 b/arch/arm/configs/rockchip_defconfig @@ -31,7 +31,10 @@ CONFIG_JUMP_LABEL=y 
 CONFIG_MODULES=y  CONFIG_MODULE_UNLOAD=y  CONFIG_PARTITION_ADVANCED=y +#
 CONFIG_MSDOS_PARTITION is not set +# CONFIG_EFI_PARTITION is not set 
 CONFIG_ARCH_ROCKCHIP=y

---
 arch/arm/configs/rockchip_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index fc39351b7e2e..530c0485fa40 100755
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -34,7 +34,9 @@ CONFIG_MODULES=y
 CONFIG_MODULE_UNLOAD=y
 CONFIG_PARTITION_ADVANCED=y
 # CONFIG_MSDOS_PARTITION is not set
+CONFIG_MSDOS_PARTITION=y
 # CONFIG_EFI_PARTITION is not set
+CONFIG_EFI_PARTITION=y
 CONFIG_ARCH_ROCKCHIP=y
 # CONFIG_SWP_EMULATE is not set
 CONFIG_ARM_ERRATA_720789=y
-- 
2.35.3

